squash_bot/core/data/dataclasses.py

```diff
@@ -4,8 +4,8 @@
 @attrs.frozen
 class User:
     id: str
-    username: str
-    global_name: str
+    username: str = attrs.field(eq=False)
+    global_name: str = attrs.field(eq=False)
```

Comment:
Easier to add a `__eq__` function rather than setting the fields as false?

Reply:
This is how `attrs` recommends doing this